src/api.js

```diff
@@ -4,20 +4,24 @@ const gamesUrl = axios.create({
   baseURL: "https://mohamed-salims-games-api.onrender.com/api",
 });
 
-export const getReviews = () => {
-  return gamesUrl.get("/reviews").then((reviews) => {
+export const getReviews = category => {
+  let params;
+  category === "all-reviews"
+    ? (params = undefined)
+    : (params = { params: { category } });

```

Comment:
You shouldn't do side-effects in ternaries, that's what if statements are for. 

This should be:
```js
let params = category === "all-reviews" ? undefined : { params: { category } }
```

A ternary is designed to evaluate to a value and you should use that value. 👍

Reply:
thx for that